modules/file-system/gke-persistent-volume/templates/gcs-pv.yaml.tftpl

```diff
@@ -13,8 +13,12 @@ spec:
     storage: ${capacity}
   accessModes:
   - ReadWriteMany
+  %{~ if mount_options != null ~}
   mountOptions:
-    - implicit-dirs
+  %{~ for key in mount_options ~}
+  - ${key}
+  %{~ endfor ~}
+  %{~ endif ~}
   csi:
     driver: gcsfuse.csi.storage.gke.io
     volumeHandle: ${bucket_name}
```

Comment:
there are some additional knobs in csi that can boost performance, so we would need to support these knobs eventually. 

1. https://cloud.google.com/kubernetes-engine/docs/how-to/cloud-storage-fuse-csi-driver-perf#metadata-
prefetch
2. https://cloud.google.com/kubernetes-engine/docs/how-to/cloud-storage-fuse-csi-driver-perf#reduce-quota-consumption

example:
```
csi:
    volumeAttributes:
      skipCSIBucketAccessCheck: "true"
      gcsfuseMetadataPrefetchOnMount: "true"
```